snap/snapcraft.yaml

```diff
@@ -1,5 +1,5 @@
 name: ros2-nav2
-version: '0.1'
+version: '1.1.17'
```

Comment:
Is it going to work here? Because we don't clone here, we pull upstream. So, by the time we merge, it might not be the version anymore. Can't we get that programatically?